### ISSUE TITLE
Card title/last ed. doesn't show readably in card details area, various browsers

### ISSUE THREAD

**Issue by annettekusma** [nextcloud/deck#2077]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**
When adding a new card or editing/viewing an existing one, the right area of the window, I'll call it the card details area, should show the card title and a line stating when the card was last modified, if I understand things correctly (I'm new to NextCloud). I'm testing NextCloud on various browsers and it always **shows only two rows of hight one (1) pixel** (might be the upper row of pixles of the corresponding letters to show). I can highlight/copy/paste those two rows and the pasting shows the title and last modified info. The rest of the card details area is fine. I get the same behaviour in Opera, Firefox, Edge and Chrome. I am using a free account on Tab. 

**To Reproduce**
1. Go to the Deck-app on your NextCloud account
2. Click on any card for the card details area to appear on the right
3. Try to locate the card title and last modified info inside the card details areas top region.
4. It is shown with only one (1) pixel in hight.

**Expected behavior**
I would expect to be able to read the card title and last modified info, or at least be able to scroll them.

**Screenshots**
![screenshot](https://user-images.githubusercontent.com/30964670/85866795-3f34f980-b7c8-11ea-8b27-b21637b01789.png)

**Client details:**
 - OS: I've tried on Windows 10, two different devices.
 - Browser: Firefox, Chrome, Opera, Edge. All currently updated (late june 2020).

**Web server:**
I'm using a free account on nc.nl.tab.digital

My apologies if this means, I should have asked Tab for advice first. 

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

**Where did you install Nextcloud from:**

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log

This is from Chrome:

```

a) html:

<aside data-v-0ede0fe3="" data-v-7018911e="" data-v-3865e8db="" id="app-sidebar-vue" class="app-sidebar">
    <header data-v-0ede0fe3="" class="app-sidebar-header">
        <a data-v-0ede0fe3="" href="#" title="close" class="app-sidebar__close icon-close">
        </a> 
<!----> 
        <div data-v-0ede0fe3="" class="app-sidebar-header__desc app-sidebar-header__desc--with-subtitle">
<!----> 
            <h2 data-v-0ede0fe3="" class="app-sidebar-header__title">
                Kalendersynkning
            </h2> 
<!----> 
            <p data-v-0ede0fe3="" class="app-sidebar-header__subtitle">
	        Ändrad: för 19 timmar sedan Skapat: för 21 timmar sedan
            </p> 
<!---->
        </div> 
<!---->
    </header> 

    <nav data-v-0ede0fe3="" class="app-sidebar-tabs__nav">...</nav>
    ...
</aside>

b) css:

element.style {
}
<style>
#content * {
    box-sizing: border-box;
}
article, aside, dialog, figure, footer, header, hgroup, nav, section {
    display: block;
}
html, body, div, span, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, code, del, dfn, em, img, q, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, dialog, figure, footer, header, hgroup, nav, section {
    margin: 0;
    padding: 0;
    border: 0;
    outline: 0;
    font-weight: inherit;
    font-size: 100%;
    font-family: inherit;
    vertical-align: baseline;
    cursor: default;
    scrollbar-color: var(--color-border-dark) transparent;
    scrollbar-width: thin;
}
user agent stylesheet
header {
    display: block;
}
body {
    background-color: var(--color-main-background);
    background-color: linear-gradient(40deg, var(--color-main-background) 0%, rgba(28, 175, 255, 1) 100%);
    font-weight: normal;
    font-size: 0.875em;
    line-height: 1.6em;
    font-family: var(--font-face);
    color: var(--color-main-text);
}
body {
    line-height: 1.5;
}
:root {
    --icon-confirm-fade-000: url(data:image/svg+xml;base64,P...=);
...
    --icon-menu-sidebar-000: url(data:image/svg+xml;base64,P...=);
Show All Properties (353 more)
}
:root {
    --color-main-text: #222;
    --color-main-background: #fff;
    --color-main-background-translucent: rgba(255, 255, 255, 0.97);
    --color-background-hover: #f5f5f5;
    --color-background-dark: #ededed;
    --color-background-darker: #dbdbdb;
    --color-primary: #0082c9;
    --color-primary-light: #e6f3fa;
    --color-primary-text: #fff;
    --color-primary-text-dark: #ededed;
    --color-primary-element: #0082c9;
    --color-primary-element-light: #17adff;
    --color-error: #e9322d;
    --color-warning: #eca700;
    --color-success: #46ba61;
    --color-text-maxcontrast: #767676;
    --color-text-light: #484848;
    --color-text-lighter: #6f6f6f;
    --image-logo: url(/apps/theming/image/logo?v=2);
    --image-login-background: url(/core/img/background.png?v=2);
    --image-logoheader: url(/core/img/logo/logo.png?v=2);
    --image-favicon: url(/core/img/logo/logo.png?v=2);
    --color-loading-light: #ccc;
    --color-loading-dark: #444;
    --color-box-shadow: rgba(77, 77, 77, 0.5);
    --color-border: #ededed;
    --color-border-dark: #dbdbdb;
    --border-radius: 3px;
    --border-radius-large: 10px;
    --border-radius-pill: 100px;
    --font-face: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
    --animation-quick: 100ms;
    --animation-slow: 300ms;
}

::-webkit-scrollbar {
    width: 6px;
    height: 5px;
}
::-webkit-scrollbar-thumb {
    background: var(--color-border-dark);
    border-radius: var(--border-radius);
}
::-webkit-scrollbar-track-piece {
    background-color: transparent;
}
::selection {
    background-color: var(--color-primary-element);
    color: var(--color-primary-text);
}


c) Console-log:

JQMIGRATE: Migrate is installed, version 1.4.1
notifications.js?v=c52f5a58-2:21 Proxying an event bus of version 1.1.2 with 0.2.1
t @ notifications.js?v=c52f5a58-2:21
main.js?v=c52f5a58-2:1082 Handlebars is deprecated: please ship your own, this will be removed in Nextcloud 20
ne @ main.js?v=c52f5a58-2:1082
notifications.js?v=c52f5a58-2:21 Notifications permissions not yet requested
4main.js?v=c52f5a58-2:1082 The escapeHTML library is deprecated! It will be removed in nextcloud 19.
ne @ main.js?v=c52f5a58-2:1082
main.js?v=c52f5a58-2:148 OC.requestToken changed esXFmHPAOE8/Nh2aXDpaoYNctKzW98DWHS8jlAJeaqw=:F4mrs0SZdDdyQGvbL347mPpk5s24lay5UW5r13suDN0=
4main.js?v=c52f5a58-2:1082 The escapeHTML library is deprecated! It will be removed in nextcloud 19.
ne @ main.js?v=c52f5a58-2:1082
deck.js?v=c52f5a58-2:1756 TypeError: Cannot read property 'mimetype' of undefined
    at deck.js?v=c52f5a58-2:1360
    at i.bt [as _l] (deck.js?v=c52f5a58-2:768)
    at i.<anonymous> (deck.js?v=c52f5a58-2:1360)
    at i.e._render (deck.js?v=c52f5a58-2:768)
    at i.r (deck.js?v=c52f5a58-2:768)
    at mn.get (deck.js?v=c52f5a58-2:768)
    at mn.run (deck.js?v=c52f5a58-2:768)
    at pn (deck.js?v=c52f5a58-2:768)
    at Array.<anonymous> (deck.js?v=c52f5a58-2:768)
    at Ke (deck.js?v=c52f5a58-2:768)
n.default.config.errorHandler @ deck.js?v=c52f5a58-2:1756
main.js?v=c52f5a58-2:148 OC.requestToken changed 6cNND5g2zsbd9zBX+3SNJqXfPiuebm8FjTJ2gXR2ZcE=:hI8jJK9vgr6QgUYWiDDsH9znbErwDANqwXM+wg0GA7A=
DevTools failed to load SourceMap: Could not parse content for https://nc.nl.tab.digital/apps/files_sharing/js/dist/collaboration.js.map: Unexpected token < in JSON at position 0
DevTools failed to load SourceMap: Could not parse content for https://nc.nl.tab.digital/apps/files_videoplayer/js/main.js.map: Unexpected token < in JSON at position 0
DevTools failed to load SourceMap: Could not parse content for https://nc.nl.tab.digital/apps/deck/js/deck.js.map: Unexpected token < in JSON at position 0
DevTools failed to load SourceMap: Could not parse content for https://nc.nl.tab.digital/core/js/dist/main.js.map: Unexpected token < in JSON at position 0
DevTools failed to load SourceMap: Could not parse content for https://nc.nl.tab.digital/apps/deck/js/collections.js.map: Unexpected token < in JSON at position 0
DevTools failed to load SourceMap: Could not parse content for https://nc.nl.tab.digital/apps/notifications/js/notifications.js.map: Unexpected token < in JSON at position 0
DevTools failed to load SourceMap: Could not parse content for https://nc.nl.tab.digital/apps/spreed/js/collections.js.map: Unexpected token < in JSON at position 0
main.js?v=c52f5a58-2:148 OC.requestToken changed /q8dPl4mrVQ5ahQMJmVN5h5dRxOmrR6NRbiw4DDKXQw=:k+NzFWl/4Sx0HGJNVSEs32dlFXLIz3LiCfn4o0m6O30=
notifications.js?v=c52f5a58-2:21 data.ocs.data is undefined or not an array
main.js?v=c52f5a58-2:148 OC.requestToken changed ELLr8YldEd7ilYbP0oHJdy2sVGbNv0UizDHfBwVCSV8=:ff6F2r4EXaav4/COocWoTlSUBgej3SlNgHCXRHwyLy4=
```

</details>

**Comment by stefan-niedermann** [comment]
Please tell us which version if Deck you are exactly using, because this seems to be a duplicate of https://github.com/nextcloud/deck/issues/1980 which has been fixed in `1.0.3`.

**Comment by stefan-niedermann** [comment]
ping @annettekusma see https://github.com/nextcloud/deck/issues/2077#issuecomment-650369919
